crates/rspack_core/src/stats.rs

```diff
@@ -500,7 +503,32 @@ impl Stats<'_> {
     });
 
     // TODO: a placeholder for concatenation modules
```

Comment:
We can remove the TODO now 😄

Reply:
I will remove it in https://github.com/web-infra-dev/rspack/pull/6799/files#diff-16aee29ffef703f493d3e45add4619f21709212d700fadbf55e8d479b2eedb37L505